modules/deploy/pages/deployment-option/cloud/security/cloud-enable-mtls-kafka-api.adoc

```diff
@@ -0,0 +1,89 @@
+= Enable mTLS Authentication for Kafka API
+:description: Use the Cloud API to enable mTLS for Kafka API connections on your Redpanda cluster.
+:page-cloud: true
+
+Redpanda Cloud supports mTLS authentication for the Kafka API. 
```

Comment:
@paulzhang97 what could we add here regarding why we added this authentication mechanism and why we might recommend it over SASL (is it SASL/SCRAM to be specific?)?